database/handlers/project.go

```diff
@@ -8,6 +8,31 @@ import (
 	"github.com/lib/pq"
 )
 
+func (client Client) GetProjectsByFilter(filterText string) ([]database.ProjectThumbnail, error) {
+	rowsRs, err := client.Query("SELECT id, name, description, programYear, programTerm FROM projects WHERE name LIKE '%' || $1 OR description LIKE '%' || $1;", filterText)
```

Comment:
Can't understand the like clauses in this query. Shouldn't it be like `name LIKE '%$1%' OR description LIKE '%$1$'`?